_content/booklets/namakkarapali-saha-sankhepayojana_anandajoti.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Namakkārapāḷi saha Saṅkhepayojanā: The Reverence Text with the Short Word-Commentary"
+authors:
+  - anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces/Namakkarapali.htm"
+drive_links:
+  - "https://drive.google.com/file/d/1QNf_DqpsSAC61PyuVc_eE96If_B_faqI/view?usp=sharing"
+course: burmese
+tags:
+  - poetry
```

Comment:
It is poetry, so that's a subcat not a tag. It's about the Buddha, so maybe that's a tag?  Also `pali-readers`